virttest/qemu_vm.py

```diff
@@ -609,7 +609,8 @@ def add_smp(devices):
         def add_nic(devices, vlan, model=None, mac=None, device_id=None,
                     netdev_id=None, nic_extra_params=None, pci_addr=None,
                     bootindex=None, queues=1, vectors=None, pci_bus='pci.0',
-                    ctrl_mac_addr=None):
+                    ctrl_mac_addr=None, indirect_desc=None, mrg_rxbuf=None,
```

Comment:
Could you explain the difference between using nic_extra_params and adding params like ctrl_vlan?

Reply:
I guess both are ok.

Reply:
If nic_extra_params can handle the new params, then we should not add them to the function. use "nic_extra_params = ",indirect_desc=on"" is enough.
better to keep vt concise

Reply:
Agree with @xiywang